sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_custom_forms_async.py

```diff
@@ -71,9 +71,18 @@ async def recognize_custom_forms(self):
                     print("...Label '{}' has value '{}' with a confidence score of {}".format(
                         field.label_data.text if field.label_data else name, field.value, field.confidence
                     ))
+                # [END recognize_custom_forms_async]
```

Comment:
Why move this up?

Reply:
I think my thought was that this sample snippet showcases custom form fields and I didn't want to detract from that with the tables code. But actually I agree, I'll move it back so it captures the whole snippet.